package.json

```diff
@@ -19,9 +19,9 @@
   },
   "homepage": "https://github.com/scality/S3#readme",
   "dependencies": {
-    "arsenal": "github:scality/arsenal#3e08bad",
+    "arsenal": "github:scality/arsenal#0c78ccb",
     "async": "~2.5.0",
-    "aws-sdk": "2.28.0",
+    "aws-sdk": "2.60.0",
```

Comment:
Upgrading to this specific version to get the `Date` field bugfix. 

https://github.com/aws/aws-sdk-js/blob/HEAD/CHANGELOG.md#2600